test/e2e/multicontainer/multicontainer_test.go

```diff
@@ -36,23 +36,27 @@ func TestMultiContainer(t *testing.T) {
 
 	clients := e2e.Setup(t)
 
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Sidecars: []string{
+			test.ServingContainer,
```

Comment:
I think the idea was to actually define the main image as `Image` still. It's not a sidecar, technicallyt.

Reply:
Yes for multi containers we planned to use `Sidecars` as per this [comment](https://github.com/knative/serving/pull/9855#issuecomment-722435650) right :thinking:

Reply:
Well Sidecars are the **additional** containers imo. There's still a "main" one, which I'd think is specified via Image.